.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["es2015"]
+}
```

Comment:
since you added the present via .babelrc, the following code is no longer necessary in webpack.config:

        query: {
          presets: ['es2015']
        }